manifests/components/images.json

```diff
@@ -1,7 +1,7 @@
 {
     "addon-resizer": "k8s.gcr.io/addon-resizer:1.8.7",
     "alertmanager": "bitnami/alertmanager:0.21.0-debian-10-r4",
-    "cert-manager": "bitnami/cert-manager:0.14.3-debian-10-r6",
+    "cert-manager": "bitnami/cert-manager:0.15.2-debian-10-r5",
     "cert-manager-acmesolver": "quay.io/jetstack/cert-manager-acmesolver:v0.14.3",
```

Comment:
@javsalgar we may also need to update this to `quay.io/jetstack/cert-manager-acmesolver:v0.15.2`, to match `bitnami/certmanager` semver

Reply:
Done